neo/SmartContract/Native/Tokens/Nep5Token.cs

```diff
@@ -0,0 +1,139 @@
+﻿using Neo.Ledger;
+using Neo.VM;
+using System;
+using System.Numerics;
+using VMArray = Neo.VM.Types.Array;
+
+namespace Neo.SmartContract.Native.Tokens
+{
+    public abstract class Nep5Token<TState> : NativeContractBase
+        where TState : Nep5AccountState, new()
```

Comment:
Just leaving an opportunity to configure this in the future with other "TState"? It works anyway.